dojo/templates/dojo/custom_html_report_finding_list.html

```diff
@@ -77,15 +82,22 @@ <h5>
                             <td>{{ finding.mitigated }}</td>
                             <td>{{ finding.mitigated_by }}</td>
                         {% endif %}
-                        {% if finding.cwe > 0 %}
+                        {% if finding.cwe %}
                             <td>
                                 <a target="_blank"
                                     href="https://cwe.mitre.org/data/definitions/{{ finding.cwe }}.html">
                                     <i class="fa fa-external-link"></i> {{ finding.cwe }}
                                 </a>
                             </td>
                         {% endif %}
-                        <td>{{ finding.id }}</td>
+                        {% if finding.cve %}
+                            <td>
+                                <a target="_blank"
+                                    href="https://nvd.nist.gov/vuln/detail/{{ finding.cve }}">
+                                    <i class="fa fa-external-link"></i> {{ finding.cve }}
+                                </a>
+                            </td>
+                        {% endif %}
```

Comment:
I don't believe the CVE field is really used anymore. Please use the vulnerability ID instead.

Reply:
Vulnerability ids were used instead

![image](https://user-images.githubusercontent.com/127124/205754850-c200f848-2b0a-46b0-bc8f-5989b0547406.png)